docker

```diff
@@ -57,3 +57,15 @@ docker login
 
 # To push a docker image into dockerhub repository:
 docker push <image-name>:<image-tag-name>
+
+# To list all networks daemon knows
+docker network ls
+
+# To create a network
+docker network create <network name>
+
+# To connect a container to a network
+docker network connect <NETWORK ID>or<NAME> <CONTAINER ID>or<NAME>
```

Comment:
```suggestion
# Connect a specific container to a network:
docker network connect "<network_id|name>" "<container_id|name>"
```

- Almost the same issue as above.
- Optional change: use TlDr page style delimiters inside placeholders.